pretraining/utils/dataset_utils.py

```diff
@@ -45,8 +45,8 @@ def __init__(
         assert (
             worldsize > rank
         ), f"Worldsize {worldsize} must be greater than rank {rank}"
-        self.state_params = []
-        self.reshard_params = []
+        self.state_params: List[str] = []
+        self.reshard_params: List[str] = []
```

Comment:
This is the type from my understanding, it could also be a `List[Any]` but I was trying to be more specific